analysis/config_global.yaml

```diff
@@ -1,3 +1,6 @@
+# GSLab Python Version
```

Comment:
That's not standard YAML and our Stata parser knows it. 

<img width="801" alt="screen shot 2018-03-02 at 10 03 10 am" src="https://user-images.githubusercontent.com/19935793/36914121-0d86ce12-1e01-11e8-990a-98f3ed486497.png">

Reply:
I see. That's weird, since yaml is supposed to play nice with 2-space or 4-space indentation. 

https://stackoverflow.com/questions/19975954/a-yaml-file-cannot-contain-tabs-as-indentation

Reply:
It works nicely on R and Python, fails on Stata, and I didn't test on MATLAB. Let's just let sleeping dogs lie.